.eslintrc.js

```diff
@@ -0,0 +1,92 @@
+    'airbnb',
+    'airbnb-typescript',
+    'airbnb/hooks',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:json/recommended',
+    'prettier',
+    'plugin:markdown/recommended',
+  ],
+  parser: '@typescript-eslint/parser',
+  plugins: [
+    '@typescript-eslint',
+    'unused-imports',
+    'promise',
+    'react',
+    'tss-unused-classes',
+  ],
+  ignorePatterns: [
+    '.eslintrc.js',
+    'package.json',
+    'rollup.config.js',
+    'tsconfig.json',
+  ],
+  parserOptions: {
+    extraFileExtensions: ['.json', '.md'],
+    ecmaVersion: 2020,
+    ecmaFeatures: {
+      jsx: true,
+    },
+    tsconfigRootDir: __dirname,
+    project: ['./tsconfig.json'],
+    sourceType: 'module',
+  },
+  rules: {
+    // TODO: Rules to review with the team
+    'tss-unused-classes/unused-classes': 'off',
+    '@next/next/link-passhref': 'off', // Our Links usually wrap MUI Buttons, which do not contain an <a> tag for passing the href.
+    '@next/next/no-img-element': 'off', // TODO(dannynelson) Use next/image instead of <img> for better performance // https://www.pivotaltracker.com/story/show/179033278
+    '@typescript-eslint/no-empty-interface': 'off',
+    '@typescript-eslint/no-inferrable-types': 'off', // No need to write like this: `let myBool: boolean = true;`
+    'jsx-a11y/alt-text': 'off',
+    'jsx-a11y/anchor-is-valid': 'off',
+    'jsx-a11y/click-events-have-key-events': 'off',
+    'jsx-a11y/iframe-has-title': 'off',
+    'jsx-a11y/label-has-associated-control': 'off',
+    'jsx-a11y/no-noninteractive-element-interactions': 'off',
+    'jsx-a11y/no-noninteractive-tabindex': 'off',
+    'jsx-a11y/no-static-element-interactions': 'off',
+    'jsx-a11y/tabindex-no-positive': 'off',
+    'no-console': 'off',
+    'promise/prefer-await-to-then': 'off',
+    'import/prefer-default-export': 'off',
+    'react/button-has-type': 'off',
+    'react/destructuring-assignment': 'off',
+    'react/function-component-definition': 'off',
+    'react/no-invalid-html-attribute': 'off',
+    'react-hooks/exhaustive-deps': 'off',
+    'react-hooks/rules-of-hooks': 'off',
+    'react/jsx-boolean-value': 'off',
+    'react/jsx-curly-brace-presence': 'off',
+    'react/jsx-fragments': 'off',
+    'react/jsx-no-bind': 'off',
+    'react/jsx-no-constructed-context-values': 'off',
+    'react/jsx-no-duplicate-props': 'off',
+    'react/jsx-no-useless-fragment': 'off',
+    'react/jsx-pascal-case': 'off',
+    'react/jsx-props-no-spreading': 'off',
+    'react/no-array-index-key': 'off',
+    'react/no-danger': 'off',
+    'react/no-unstable-nested-components': 'off',
+    'react/no-unused-prop-types': 'off',
+    'react/require-default-props': 'off',
+    'react/self-closing-comp': 'off',
+    'react/state-in-constructor': 'off',
+    'spaced-comment': 'off',
+    'no-restricted-exports': 'off',
+
+    // Approved rules that we use to match the codebase to UD code style
+    'react/display-name': 'off',
+    'react/prop-types': 'off',
+  },
```

Comment:
Since we're starting a new repo I think we can simply use the airbnb linter since that's what we're aiming for.

Reply:
Removed all the rules from eslint config except for the `no-restricted-exports` since it's interfering with `export {default} from './Component'` code (such code is also present in Material UI repo as well)

Do we want to minimize the `extends` and `plugins` too, or was it only about avoiding rules offing?

Reply:
It was only about avoiding rules offing. The rest sgtm for now.

Reply:
Got it, thanks, resolving for now